python/tchannel/req_handler.py

```diff
@@ -0,0 +1,67 @@
+class RequestHandler(object):
+
+    def __init__(self):
+        pass
+
+    def handle_request(self, context, conn):
+        raise NotImplementedError
```

Comment:
`NotImplementedError()`.

Without the brackets, you're trying to throw the class itself. Not sure if that's a throwable.